client/my-sites/plugins/controller.js

```diff
@@ -164,7 +178,19 @@ controller = {
 		 * if no site URL is provided, bail if a WordPress.com filter was provided.
 		 * Only Jetpack plugins should work when no URL is provided.
 		 */
-		if ( siteUrl && ( ( ! site.jetpack && ! includes( [ 'all', wpcomFilter ], appliedFilter ) ) || ( site.jetpack && appliedFilter === wpcomFilter ) ) ) {
+		if (
+			siteUrl &&
+			(
+				(
+					! site.jetpack &&
+					! includes( [ 'all', wpcomFilter ], appliedFilter )
+				) ||
+				(
+					site.jetpack &&
+					appliedFilter === wpcomFilter
+				)
+			)
```

Comment:
Spacing looks a bit crazy here. Let's add a few more variables to shorten this up.